### PR TITLE
max open connection in database updated from 1000 to 100

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Decrease the number of max open connection to 100, 1000 is very high number for default

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/2171
## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by pmahindrakar-oss — APPROVED** [review]

**Review by katrogan — APPROVED** [review]

**Review by EngHabu — APPROVED** [review]

**Comment by honnix** [comment]
Shall we fix https://docs.flyte.org/en/latest/deployment/cluster_config/flyteadmin_config.html#maxidleconnections-int as well? Thanks.

**Comment by honnix** [comment]
Screenshot 2022-06-07 at 09 23 43
So the sql execution is slow.

**Comment by honnix** [comment]
We could try to give cloudsql-proxy more CPU. Maybe double the current allocation.

**Comment by honnix** [comment]
_Sorry I accidentally commented on wrong PR. I have removed my other comments._

**Comment by honnix** [comment]
> @honnix are you seeing any side-effects with the current idle connections set to 10? hopefully 10 is low enough to not consume excessive resources

No I meant the doc still says 1000 as default max open connections.

**Comment by katrogan** [comment]
ah @honnix I think we need to upgrade flytestdlib in [flyteadmin](https://github.com/flyteorg/flyteadmin/blob/master/go.mod#L19) with this fix (https://github.com/flyteorg/flytestdlib/releases/tag/v1.0.3)

**Comment by honnix** [comment]
Right I see. It's not a problem for us since we have custom configuration anyway. Thanks for fixing it.
